src/foremast/app/create_app.py

```diff
@@ -85,13 +86,22 @@ def create_app(self, pipeline_configs=None):
         Raises:
             AssertionError: Application creation failed.
         """
+        self.pipeline_configs = pipeline_configs
```

Comment:
I think this should be moved into `__init__()`. I find it strange to create a new property in some random method. I think it should be passed in when you instantiate `SpinnakerApp` class.